CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ initialize_component_helpers(aikido)
 include(CMakePackageConfigHelpers)
 include(FindPkgConfig)
 
-find_package(Boost REQUIRED COMPONENTS filesystem)
+find_package(Boost REQUIRED COMPONENTS filesystem graph program_options system)
```

Comment:
I haven't had a chance to actually review this PR (or the corresponding PR in `pr-ompl`), but I think we should keep these kind of dependencies _out_ of AIKIDO if they're only needed in `pr-ompl`.

As I think you were saying offline, I think it'd be best if this was an optional dependency on `pr-ompl`, rather than requiring that `pr-ompl` be bundled with AIKIDO. It seems that having this organization where we add our methods directly to `planner/ompl/Planner.cpp` wouldn't quite work in that scenario? I feel like we'd need to have them in a separate `PRPlanner` file that we can choose to compile depending on whether `pr-ompl` is actually available, but maybe someone who's more familiar with CMake can weigh in.

Reply:
This dependency is not required in AIKIDO actually, my bad - removed them in a recent commit. I currently have `pr-ompl` as an optional dependency but yes, moving `pr-ompl` methods out of `planner/ompl/Planner.cpp` will allow `OMPL` and `pr-ompl` dependencies to be separate.